### PR TITLE
Version: 3.33.1

### PR DESCRIPTION
**Pull request by kira-syslogng**
Tarball is available at: https://github.com/syslog-ng/syslog-ng/actions/runs/1001876314

TODO:

- [x] #3689
- [x] #3703
- [x] #3715
- [x] Rebase + remove news file fragments

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by Barkodcz — DISMISSED** [review]

**Comment by MrAnno** [comment]
Waiting for Docker Hub rebuild.

File: NEWS.md [hunk 1]
```diff
@@ -1,101 +1,133 @@
...
-   Workers are based on the connection pooling feature of the MongoDB C library.
-   
-   This increases the throughput of the MongoDB destination driver.
-   
-   Example:
-   
+ * MQTT destination
+
+   The new `mqtt()` destination can be used to publish messages using the MQTT protocol.
+   Currently MQTT 3.1.1 and 3.1 are supported, 5.0 and SSL support will be added later.
```
Comment (Kokan) [inline NEWS.md:9]: I do not think that we have to mention 5.0 and SSL support, this reads like we commit to support those. I would simply either not mention them, or note that those are not yet supported.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md (hunk 1 above)
Comment (MrAnno) [reply NEWS.md]: Okay, I added a note instead.

**Review by Kokan — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (Kokan) [reply NEWS.md]: Thanks.

**Review by szemere — APPROVED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**